docs/configuring-metabase/environment-variables.md

```diff
@@ -348,6 +348,15 @@ Default: `null`
 
 The port for [MB_DB_HOST](#mb_db_host).
 
+
+### `MB_DB_QUERY_TIMEOUT_MINUTES`
+
+Type: integer<br>
+Default: `180`
```

Comment:
This is wrong, but I guess it doesn't matter since this file gets autogenerated anyway